types/office-js-preview/index.d.ts

```diff
@@ -15569,24 +15569,95 @@ declare namespace Office {
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Message Compose
+         * 
+         * **Important**: The `closeAsync` method is only available for preview in Outlook on Windows.
+         * 
+         * **Errors**:
+         * 
+         * - `The operation was cancelled by the user`: The user selects **Cancel** from the save dialog and the `discardItem` property isn't defined or is set to `false`.
+         * 
+         * - `The operation is not supported`: The `closeAsync` method attempts to close an inline reply or existing draft and the `discardItem` property isn't defined or
+         * is set to `false`.
+         * 
+         * @param options - An object literal that contains one or more of the following properties:-
+         *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
+         *        `discardItem`: If `true`, the current message being composed is closed and unsaved changes are discarded. When the parameter isn't declared or is
+         *        set to `false`, a save dialog appears for the new message being composed prompting the user to save a draft, discard changes, or cancel the operation.
+         * 
+         * @param callback - Optional. When the method completes, the function passed in the callback parameter is called with a single parameter,
+         *                `asyncResult`, which is an `Office.AsyncResult` object.
+         * 
+         * @beta
+         */
+        closeAsync(options: Office.AsyncContextOptions & { discardItem: boolean }, callback?: (asyncResult: Office.AsyncResult<void>) => void): void;
+        /**
+         * Closes the current message being composed. The message being composed can be a new message, an inline reply, or an existing draft.
+         * 
+         * The behavior on a new message being composed depends on whether the message contains any unsaved changes. If no changes have been made, the message is
+         * closed without a save dialog. On the other hand, if the message contains unsaved changes, a save dialog appears prompting the user to save a draft,
+         * discard changes, or cancel the operation.
+         *
+         * @remarks
+         * [Api set: Mailbox preview]
+         * 
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Message Compose
+         * 
+         * **Important**: The `closeAsync` method is only available for preview in Outlook on Windows.
```

Comment:
I think what's meant is that it is only available in Windows and it is in preview. The current wording makes it sound like it is in preview on Windows, but released on other platforms.

Reply:
Good point. Reworded.